lib/apiservers/engine/backends/image.go

```diff
@@ -184,6 +155,7 @@ func (i *Image) PullImage(ctx context.Context, ref reference.Named, metaHeaders
 		return err
 	}
 
+	layerCache.Update()
```

Comment:
That's actually at the bottom of the `PullImages` function (I think your diff collapsed it making it appear like it was in `Images`).